src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/packages.config

```diff
@@ -5,7 +5,7 @@
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.OperationalInsights" version="0.15.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.OperationalInsights" version="0.16.0-preview" targetFramework="net45" />
```

Comment:
You are referring to older versions of Microsoft.IdentityModel.Clients.ActiveDirectory (2.28.0) and Microsoft.Rest.ClientRuntime.Azure.Authentication (2.2.6-preview) in this file - please update to 2.28.1 and 2.2.7-preview respectively, in your packages.config and csproj for both your cmdlets project and your test project